src/main/resources/hibernate.cfg.xml

```diff
@@ -13,6 +13,11 @@
         <property name="show_sql">true</property>
         <property name="hbm2ddl.auto">update</property>
 
+        <mapping class="core.basesyntax.model.Comment"/>
+        <mapping class="core.basesyntax.model.Message"/>
+        <mapping class="core.basesyntax.model.MessageDetails"/>
+        <mapping class="core.basesyntax.model.Smile"/>
+        <mapping class="core.basesyntax.model.User"/>
```

Comment:
```suggestion
        <mapping class="core.basesyntax.model.Comment"/>
        <mapping class="core.basesyntax.model.Message"/>
        <mapping class="core.basesyntax.model.MessageDetails"/>
        <mapping class="core.basesyntax.model.Smile"/>
        <mapping class="core.basesyntax.model.User"/>
```
we shouldn't change this file 
https://mate-academy.github.io/jv-program-fulltime/03_hibernate/checklist/hibernate_02.html

Reply:
Mapping is nessesary, i'm 100% sure about it and heard it from Sofia on QnA. Also i heard, as one of student said that project without mapping also pass tests.